print-numbers-server/index.js

```diff
@@ -0,0 +1,23 @@
+//Call the http module and define several constants (http,host,port) 
+const http = require("http");
+const host = "127.0.0.1";

```

Comment:
PORT and HOST must be in uppercase because are static constants.

Reply:
Done